<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="input_pick_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wähle Eingabedatei aus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_pick_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wähle Ausgabedatei aus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_workflow_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_file_path_line_edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_file_path_line_edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="state_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: Waiting...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="groups_spinbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="iterations_spinbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="groups_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl Gruppen:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iterations_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl Iterationen: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
